web/src/copy_messages.ts

```diff
@@ -394,3 +378,11 @@ function get_end_tr_from_endc($endc: JQuery<Node>): JQuery {
 
     return $endc.parents(".selectable_row").first();
 }
+
+export function initialize(): void {
+    document.addEventListener("copy", (ev) => {
+        if (copy_handler(ev)) {
+            ev.preventDefault();
+        }
+    });
```

Comment:
I worry that this is no longer scoped to the message feed, like the original `hotkey.js` code was, and so might do weird things when copying in random text areas or other parts of the app.

Reply:
I think my current thinking is that the `hotkey` code was only limited to the message feed being present in the center pane -- but the handler must have already been debugged to handle copying random other elements. 

I think it's probably correct to not limit it -- this would help copying from the compose preview area with the inbox open, for example.